### DISCUSSION TITLE
add a hash file.txt with next releases

### DISCUSSION THREAD

**Discussion by olumolu** [Jeffser/Alpaca#211] [General]
Due to the reason cryptographically signed flatpak does not offer the security.
But providing a hash make it a good way so users can check the integrity of the file.
Blake3 hash is the advance one that i recommend still you can provide sha.

**Comment by Jeffser** [comment]
you mean a hash for the flatpak package inside the github release?

**Reply by olumolu** [reply]
I mean with the GitHub release 
File.flatpak
Filehash.txt
Containing a hash of the application those want to check the integrity of the file.

**Reply by Jeffser** [reply]
oh yeah I included that on this update

**Reply by olumolu** [reply]
Blake3 or sha256 can be good option.
Though blake3 is advance but still this is okey to have any of them

**Reply by Jeffser** [reply]
sha256 is enough, people can even build their own flatpak package, so I don't think it needs much more than that

**Reply by olumolu** [reply]
Sure
